x-pack/plugins/lens/public/heatmap_visualization/suggestions.test.ts

```diff
@@ -357,6 +357,59 @@ describe('heatmap suggestions', () => {
       ]);
     });
 
+    test('for tables with a single metric dimension', () => {
+      expect(
+        getSuggestions({
+          table: {
+            layerId: 'first',
+            isMultiRow: true,
+            columns: [
+              {
+                columnId: 'test-column',
+                operation: {
+                  isBucketed: false,
+                  dataType: 'number',
+                  label: 'Count of records',
+                },
+              },
+            ],
+            changeType: 'reduced',
+          },
+          state: {
+            layerId: 'first',
+            layerType: layerTypes.DATA,
+          } as HeatmapVisualizationState,
+          keptLayerIds: ['first'],
+        })
+      ).toEqual([
+        {
+          state: {
+            layerId: 'first',
+            layerType: layerTypes.DATA,
+            shape: 'heatmap',
+            valueAccessor: 'test-column',
+            gridConfig: {
+              type: HEATMAP_GRID_FUNCTION,
+              isCellLabelVisible: false,
+              isYAxisLabelVisible: true,
+              isXAxisLabelVisible: true,
+              isYAxisTitleVisible: false,
+              isXAxisTitleVisible: false,
+            },
+            legend: {
+              isVisible: true,
+              position: Position.Right,
+              type: LEGEND_FUNCTION,
+            },
+          },
+          title: 'Heat map',
+          hide: true,
```

Comment:
Is this line the crux of the test? If so, paring down the assertion would make the test's intent clearer.